.storybook/themes/news.scss

```diff
@@ -201,10 +201,7 @@
 							"inline-size": 100%,
```

Comment:
This may need to be updated again once https://github.com/WPMedia/arc-themes-blocks/pull/1823 is merged in. I expect their may be some conflicts.